### DISCUSSION TITLE
Feature Request: backup description text

### DISCUSSION THREAD

**Discussion by snpefk** [ankitpokhrel/jira-cli#512] [Ideas] [answered]
It happened with me several times: VPN lags and I lose detailed comment or some required field missing and description of the issue lost forever. 

I suggest to buffer everything that user written with editor. Nothing fancy, no automation, just backup text somewhere. Maybe save them in `/tmp/jira-scrach` or `$XDG_STATE_HOME/jira-scratch`?

**Comment by ankitpokhrel** [comment, marked as answer]
I think this is similar to #490

**Reply by snpefk** [reply]
Hmm, yeah, look similar. Should I port my suggestion about `/tmp` / `$XDG_STATE_HOME` for historical reason? Or you already have clear vision about implementation.

**Reply by ankitpokhrel** [reply]
Yes, feel free to do so. Any form of ideas are welcome :)
